### ISSUE TITLE
Consider adding overlay support to API

### ISSUE THREAD

**Issue by brendan-ward** [consbio/mbgl-renderer#5]
Could be similar to the Mapbox static API: https://www.mapbox.com/api-documentation/#retrieve-a-static-map-from-a-style

**Comment by brendan-ward** [comment]
This can be done directly through the style: https://www.mapbox.com/mapbox-gl-js/style-spec#sources-geojson

So we just need to add an example to demonstrate it.

**Comment by normanrz** [comment]
I agree that this doesn't need a separate API. There is an example in the tests now: https://github.com/consbio/mbgl-renderer/blob/master/tests/fixtures/example-style.json

I think only a section in the Readme is missing.
